### PR TITLE
Enables Dead OOC at round-end

### LINKED ISSUE #29321: OOC turned off for dead mobs doesn't turn back on when the round ends
Global OOC mutes do this, so should OOC mutes for dead mobs.

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
Fixes #29321

[Changelogs]:
[]: 

:cl: Naksu
fix: OOC for dead people is now (re-)enabled at round-end
/:cl:

[why]: 
Consistency with global OOC, which is also enabled at round-end

### CODE REVIEW

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -135,6 +135,7 @@ SUBSYSTEM_DEF(ticker)
 			if(!mode.explosion_in_progress && mode.check_finished(force_ending) || force_ending)
 				current_state = GAME_STATE_FINISHED
 				toggle_ooc(1) // Turn it on
+				GLOB.dooc_allowed = TRUE // Previous line already announces OOC availability, also DOOC has no convenience procs anyways
```
Comment (optimumtact) [inline code/controllers/subsystem/ticker.dm]: why not make one

Comment (vuonojenmustaturska) [reply code/controllers/subsystem/ticker.dm]: toggle_ooc is only called here and and when parsing the config, toggle_dooc would only be called here to do exactly what this one line does and nothing more.

Comment (optimumtact) [reply code/controllers/subsystem/ticker.dm]: why not make one

**Comment by lordpidey** [comment]
I'm having a slight immature giggle over the variable name "dooc".

Aright, immature giggle over.

This sounds like a good change.

**Comment by BeeSting12** [comment]
@lordpidey 
lol dooc. hehe

**Comment by YakumoChen** [comment]
we have a dead ooc? You mean just deadchat? People disable that??

**Comment by vuonojenmustaturska** [comment]
No, there's a separate global to disable ooc chat for dead people, as it's the only way ghosts can talk with alive people. The "problem" is that it doesn't re-enable with regular OOC chat.

File: code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/controllers/subsystem/ticker.dm]: make it a proc Just In Case

**Review by Cyberboss — CHANGES_REQUESTED** [review]
In case it wasn't obvious, make the proc

**Review by Cyberboss — CHANGES_REQUESTED** [review]
https://github.com/tgstation/tgstation/blob/e5929eb15bd5815ac92a3d750fc74ca3192e36de/code/modules/admin/admin.dm#L516

Make this use that proc

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**